Android-Sample/README.md

```diff
@@ -16,217 +20,108 @@ The sample app only subscribes to the test stream. It does not subscribe to othe
 
-   For a list of supported devices, see the "Developer and client requirements"
-   on [this page](https://tokbox.com/developer/sdks/android/).
+    ```java
+    NetworkQualityTest networkQualityTest = new NetworkQualityTest(
+        this,  // Current activity
+        networkQualityTestConfig,  // Configuration object
+        networkQualityTestCallbackListener  // Callback listener
+    );
+    ```
 
- The app uses a test stream and session to determine the client's ability to publish a stream
- that has audio and video. At the end of the test it reports one of the following:
+3. **NetworkQualityTestConfig**
 
-   * You're all set -- your client can publish an audio-video stream.
+    The `NetworkQualityTestConfig` class is used to configure the parameters of the network quality test. It provides the following constructor:
 
-   * Voice-only -- Your bandwidth is too low for video.
+    You can configure the following parameters:
+    
+    - `sessionId`: The session ID of your OpenTok session.
+    - `apiKey`: Your OpenTok API key.
+    - `token`: The token for connecting to the session.
+    - `resolution`: The camera capture resolution for the publisher. Defaults to `HIGH` if not specified.
+    - `testDurationSec`: The duration of the test in seconds. Defaults to `30 sec` if not specified.
 
-   * You can't connect successfully -- Your bandwidth is too low for audio or video.
+4. **Handle Callbacks**
 
-## Understanding the code
+    Once the test is running, you'll receive callbacks in the provided listener methods:
+    
+    - `onQualityTestResults`: Receive the final test results and the recommended setting based on quality thresholds.
+    - `onQualityTestStatsUpdate`: Receive real-time statistics updates during the test.
+    - `onError`: Handle errors that occur during the test.
 
-The MainActivity class connects to the test OpenTok session. Upon connecting to the session,
-the app connects initializes an OpenTok Publisher object it uses to test stream quality.
-Upon publishing, the app subscribes to the test stream it publishes:
-
-```java
-@Override
-public void onStreamCreated(PublisherKit publisherKit, Stream stream) {
-    Log.i(LOGTAG, "Publisher onStreamCreated");
-    if (mSubscriber == null) {
-        subscribeToStream(stream);
-    }
-}
-```
-
-The View object representing the the publisher and subscriber videos are not added to the app's
-view tree, so the test video is not displayed.
-
-When subscribing to the stream, the app calls the `setVideoStatsListener(videoStatsListener)`
-method of the Subscriber object. 
-
-```java
-private void subscribeToStream(Stream stream) {
-    mSubscriber = new Subscriber(MainActivity.this, stream);
+    Here an example of ```NetworkQualityTestCallbackListener```:
+    ```java
+    new NetworkQualityTestCallbackListener() {
+        @Override
+        public void onQualityTestResults(String recommendedSetting) {
```

Comment:
Can the `recommendedSetting` be `do not use video` or something similar? If the minimum conditions are not met?